### DISCUSSION TITLE
Static Sign Out button someday?

### DISCUSSION THREAD

**Discussion by AwardsNetwork** [directus/directus#16514] [Feature Requests]
Hello! 

We are new to using Directus and our team is just starting to get familiarized with it. So far, several of them have commented about having a hard time finding the sign out button since it doesn't appear until you hover over the user icon. Since we will be rolling out usage to a lot of our clients in the near future, we just wanted to inquire as to whether this might become its own static button on the menu someday. Until then, we will just make sure we cover it in training, but wanted to throw the idea out there. 

Thanks in advance for any consideration! :)

**Comment by rijkvanzanten** [comment]
Thanks for the suggestion! Lets see how much traction this gets in upvotes etc to see how much of the community agrees and we can make this switch if appropriate 🙂

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
